initAuth.js

```diff
@@ -0,0 +1,43 @@
+import { init } from "next-firebase-auth";
+
+const initAuth = () => {
+  init({
+    authPageURL: "/",
+    appPageURL: "/auth",
+    loginAPIEndpoint: "/api/login", // required
+    logoutAPIEndpoint: "/api/logout", // required
+    // firebaseAuthEmulatorHost: "localhost:9099",
+    // Required in most cases.
+    firebaseAdminInitConfig: {
+      credential: {
+        projectId: process.env.FIREBASE_PROJECT_ID,
```

Comment:
these are .env.local variables as these are private keys (so its git ignored).  if this PR is approved please add the below values to a `.env.local` file relative to root for this project -> `lms-platform/.env.local` (relative to root meaning in the same directory as the package.json, tailwind.config.js, etc...

.env.local values
`FIREBASE_PROJECT_ID=lms-project-x691
FIREBASE_CLIENT_EMAIL=firebase-adminsdk-1vysr@lms-project-x691.iam.gserviceaccount.com
FIREBASE_PRIVATE_KEY=-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDSf/2luvcDIERB\nbW5aLcWAUU+XHiDaLM8q79u1vIp199cItF/Z1mHB7aVx19or+BS//rs5wiVPFAAN\nESBQ/TYVAG1tmJ17VV1sr2dwJ6dFWVhne2/7T64JVWU+b5eNrI3qMy9zZMOGUZOp\nLE6Amo2SO6ggfKs1oYP0JGKRqeIhrHhyNDcRGOv6kxByDBG0ngreTAPlcCS4OH0A\nKyHyult4yVavO+6+moc4eCHIP8m5oNG3LwmQPkYpah8afRAdEDcZ7aPDh+YDRRo+\nQFUHs911gRhQKfgytETNaWPnPuSghsQHTUBeIR9ZJHdV+w44j+WHOSW56kGsXL9R\nRkKo1+XxAgMBAAECggEAAPsUD3LkVPM1mYnbrV+DGQC6po8/MJBhbPoZ/5yrtxE0\nUxk0N4ruBiH6rvc2MCrMox4fXn1rmx99C6fftbGPVY2CD67vSr3BpMiZWm08icL+\nPvmvvR0AfZDvJZluIadVxYXZvSThcRbVrFCCuLWiTnBjwrZ9Uvo1conTRynX02e0\n1qc3VWzyrFEFTOzkhmx/+Vf0jJ/3DVv/9As51auN90LEbSSgRkl/oYZBbrJNGnwg\nkpzj/qAbh+iyx1W5hKCBnMJlJcQZ6dF9ZXcILZsUL0saV2KQc+HNrXYq8RT8yPxh\nRXA8xM69QNvLdOwDgDOhRMkGWex9nNR93uLXYQxMgQKBgQDxlbTlqQg+i3s4TwST\nbu+vans2Tqw7EXR6r0zAE14rp2wVNEeZvbuaKKlLxikBner6tFdzjIf+53yimxHi\nX/cKxpDW8H04Jekrq40dpt/YhVbD8NDoR2iqAxFFqp6vG3hsTQVltjueBykex6NP\nzUmTw6ldAYL+Fqc+Un5yCgWHcQKBgQDfD3PRoNaXB8avraKZM5Uv3bggQipFxl7c\n61znS6e8WOo9+ZMHIAu5rL+N7lTHe/JvC0Gwg7jxHllossQd6qEe1CalZALThOMq\nkmkuE1xBaHCjyIUx/O5SIvCA0MMuyGOWq56S9HYAz29jV9IxesMjoySjOpaRBbx2\nN2KNLAIGgQKBgQDgukaArPDCXbXN95NmInhMUV00c5K7j80uowteUaX0LdXZE7zX\nQyvUh6aUmk1i96eydzgygLOxZ10FVgMY9fQ0zenGSAfzxzBUM00eHDF2lPLVapqq\nuGKgZCIZEFJ0ybUH6c3vg8mJg1e5DjTY8zpSflx57AISorgIvWxqYPArwQKBgDow\n5M0WQYUrESk5igafj1bqZ6N6v15ujsbhW943PW4EYC+E0Vd1g7mPmHVlbJaR/j/7\n1so+wtucSETpBgKr+BZ70A0rlxUpQL4Vbf0ILfQeU+xWXnEZA+9HkCh6zcCBpIFl\nCjxaLAKwGuX2vi4MUaYVZHR0z1zL+0t3zWz4wxqBAoGAVoD7wlOUTiLRS1I1oqqG\nkWKFtyGP8g+FuSrZA2NIbgQ/XhBJFShMSlFITPEzcRHVPpLWdJKaqfNqyvcom0bc\nOX4z692LNJcWz5KnnVs4RWoFBV4MR1P/uK3CeQUW3nmHmrJZVq3J5fPOCpPQnBYz\n3cBXvcmS5kP7EB1kvMc3lPY=\n-----END PRIVATE KEY-----\n
`